### DISCUSSION TITLE
2024-05-28(화)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#51] [데일리스크럼]
아래 양식 복사해서 댓글로 작성해주시면 됩니다😃
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by minyoung0503** [comment]
### 완료한일
* 리뷰 박스 컴포넌트 구현 

### 해야할 일
* textarea :  value에 따라 동적으로 크기가 변하도록 만들기

### 어려운 일
* textarea :  value에 따라 동적으로 크기가 변하도록 만들기

**Comment by Seoin02** [comment]
### 완료한일
- 카카오톡 OAuth 권한 신청

### 해야할 일
- 구글 OAuth 생성
- 카카오톡 OAuth 로직 짜기

### 어려운 일
- 카카오톡 OAuth 닉네임과 이메일은 권한 심사없이 받을 수 있는데, 이름은 권한 심사 결과가 3-5일 나온다고 합니다!

**Comment by q45402sk** [comment]
### 완료한일
카드 컴포넌트 거의 완성

### 해야할 일
카드 컴포넌트 완성 
찜 컴포넌트 시작

### 어려운 일
낙관적 업데이트 적용이 어려울 것 같습니다

**Comment by devwqc** [comment]
### 완료한일
- tanstack-react-query 설치 및 설정
- Layout 작업 진행

### 해야할 일
- Layout 작업

### 어려운 일
- 앱 화면을 고정으로 데스크탑에서 보여주려고 하니까 어려움이 있네요

**Comment by seolsis** [comment]
### 완료한일
- 결제부분 변수로 수정
- 선택 항목에 따른 결제액 변경 로직 구현

### 해야할 일
- 장바구니 css
- 결제부분 컴포넌트 분리 (결제페이지에서도 사용됨)

### 어려운 일
- 수량에 따른 금액 실시간 변경
